doc/deployment_guide.md

```diff
@@ -0,0 +1,806 @@
+
+```yml
+  hyperswarm:
+    build:
+      context: .
+      dockerfile: Dockerfile.hyperswarm
+    image: keychainmdip/hyperswarm-mediator
+    environment:
+      - KC_NODE_NAME=${KC_NODE_NAME}
+      - KC_GATEKEEPER_URL=${KC_GATEKEEPER_URL}
+    volumes:
+      - ./data:/app/hyperswarm/data
+    user: "${KC_UID}:${KC_GID}"
+    depends_on:
+      - gatekeeper
+```
+
+#### registry services (TBTC)
+
+MDIP uses immutable ledgers to record evidence of a decentralized consensus on the order of operations that form the history of a DID. The MDIP protocol is blockchain agnostic and provides 2 Testnet examples to demonstrate the protocol's portability: Bitcoin Testnet4 and Feathercoin Testnet. Each DID is registered on a blockchain selected by the end-user at the time of creation. DID operations are batched and exported by the MDIP Mediator to its associated blockchain node.
+
+In the example below, we use a generic bitcoin-code node configured to operate on testnet4. A similar service configuration block for Feathercoin Testnet is provided in the `kc` repository.
+
+```yml
+  tbtc-node:
+    image: macterra/bitcoin-core:v27.99.0-2f7d9aec4d04
+    volumes:
+      - ./data/tbtc:/root/.bitcoin
+
+  tbtc-mediator:
+    build:
+      context: .
+      dockerfile: Dockerfile.sat
+    image: keychainmdip/satoshi-mediator
+    environment:
+      - KC_GATEKEEPER_URL=${KC_GATEKEEPER_URL}
+      - KC_NODE_ID=${KC_NODE_ID}
+      - KC_SAT_CHAIN=TBTC
+      - KC_SAT_NETWORK=testnet
+      - KC_SAT_HOST=tbtc-node
+      - KC_SAT_PORT=48332
+      - KC_SAT_START_BLOCK=${KC_TBTC_START_BLOCK}
+      - KC_SAT_USER=${KC_TBTC_USER}
+      - KC_SAT_PASS=${KC_TBTC_PASS}
+      - KC_SAT_WALLET=${KC_TBTC_WALLET}
+      - KC_SAT_IMPORT_INTERVAL=${KC_TBTC_IMPORT_INTERVAL}
+      - KC_SAT_EXPORT_INTERVAL=${KC_TBTC_EXPORT_INTERVAL}
+      - KC_SAT_FEE_MIN=${KC_TBTC_FEE_MIN}
+      - KC_SAT_FEE_MAX=${KC_TBTC_FEE_MAX}
+      - KC_SAT_FEE_INC=${KC_TBTC_FEE_INC}
+    volumes:
+      - ./data:/app/satoshi/data
+    user: "${KC_UID}:${KC_GID}"
+    depends_on:
+      - tbtc-node
+```
+
+### Install Local CLI Dependencies
+
+NodeJS dependencies should be resolved both locally on the Host and within the Docker containers. Dependencies resolved on the host machine enables command-line admin tools and scripts. The following command will launch the `clean` installation of all npm package dependencies for MDIP.
+
+```bash
+npm ci
+```
+
+## Launching and Running an MDIP Node
+
+Once the server requirements and dependencies are resolved, the next step will create and launch the docker containers.
+
+```bash
+./start-node
+```
+
+On the first run, the `start-node` command will take up to 15 minutes to prepare the docker containers needed by the MDIP node. This will include Bitcoin Testnet and Feathercoin Testnet as registries. 
+
+The script contains 2 commands:
+
+```bash
+docker compose build "$@"
+docker compose up "$@"
```

Comment:
BTW, consider adjusting the script to launch with `-d` so that the node isn't tied to the current terminal session. You can add `docker compose logs -f` to get the same running output.